res/values/strings.xml

```diff
@@ -120,6 +121,9 @@
     <string name="contact">Contact</string>
     <string name="verified_contact">Verified contact</string>
     <string name="camera">Camera</string>
+    <string name="camera_capture_description">Capture</string>
+    <string name="camera_change_description">Change camera</string>
```

Comment:
Maybe "switch" instead of "change"? Sounds nicer to me

Reply:
I just take that from Signal, but agree that switch sounds better +1